docker/ubuntu/18.04/cpu_ispc_build/Dockerfile

```diff
@@ -1,9 +1,9 @@
 #
-#  Copyright (c) 2017-2024, Intel Corporation
+#  Copyright (c) 2017-2025, Intel Corporation
 #
 #  SPDX-License-Identifier: BSD-3-Clause
 
-ARG LLVM_VERSION=17.0
+ARG LLVM_VERSION=20.1
```

Comment:
It seems that we just forgot to remove LLVM version when we were moving to superbuild. Let's just remove it (since it's coming from presets) to avoid these updates in the future.

Reply:
Please remove LLVM version and feel free to merge the PR.

Reply:
It looks like we use `LLVM_VERSION` parameter of ubuntu 22.04 Dockerfile. Same parameter of ubuntu 18.04 docker can be removed.